Client/Frontend/BraveVPN/BraveVPNSettingsViewController.swift

```diff
@@ -141,6 +148,21 @@ class BraveVPNSettingsViewController: TableViewController {
     
     deinit {
         NotificationCenter.default.removeObserver(self)
+        serverListRequest?.cancel()
+        serverListRequest = nil
+    }
+    
+    private var serverListRequest: URLSessionDataTask?
+    
+    private func fetchRegionList() {
+        BraveVPN.requestAllServerRegions() { [weak self] regionList in
+            guard let regionList = regionList else {
+                log.error("Failed to fetch vpn region list")
+                return
+            }
```

Comment:
Shouldn't we show an error to the user?

Reply:
Not at this point, we show it after user taps on 'Change location' cell.
It's kind of prefetch, called when VPN settings screen is opened, 
so when user taps on 'Change location' cell, next screen will have needed data without having to wait, or error alert will be shown if it failed